src/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2328,6 +2331,7 @@
   </data>   
   <data name="PlatformNotSupported_StrongNameSigning" xml:space="preserve">
     <value>Strong-name signing is not supported on this platform.</value>
+
```

Comment:
Yes sorry I was trying to merge a conflict from github UI. I will do the merge from my console and rebase.